src/parser/parser.yy

```diff
@@ -856,14 +858,32 @@ order_by_sentence
     ;
 
 fetch_vertices_sentence
-    : KW_FETCH KW_PROP KW_ON name_label vid_list yield_clause {
+    : KW_FETCH KW_PROP KW_ON fetch_labels vid_list yield_clause {
         $$ = new FetchVerticesSentence($4, $5, $6);
     }
-    | KW_FETCH KW_PROP KW_ON name_label vid_ref_expression yield_clause {
+    | KW_FETCH KW_PROP KW_ON fetch_labels vid_ref_expression yield_clause {
         $$ = new FetchVerticesSentence($4, $5, $6);
     }
-    | KW_FETCH KW_PROP KW_ON MUL vid {
-        $$ = new FetchVerticesSentence($5);
+    | KW_FETCH KW_PROP KW_ON MUL vid_list yield_clause {
+        auto labels = new nebula::FetchLabels();
+        labels->addLabel(new std::string("*"));
+        $$ = new FetchVerticesSentence(labels, $5, $6);
+    }
+    | KW_FETCH KW_PROP KW_ON MUL vid_ref_expression yield_clause {
```

Comment:
If you want to support multi props,  maybe we need to support "KW_PROP" and "KW_PROPS" both.

Reply:
just like value/values...